docs/content/latest/yugabyte-cloud/release-notes.md

```diff
@@ -15,6 +15,13 @@ showAsideToc: true
 
 ## Releases
 
+### December 16, 2021
+
+#### New features
+
+- Self service Virtual Private Cloud (VPC) networking. Use VPC networks to lower network latencies, make your application and database infrastructure more secure, and reduce network data transfer costs. Create VPCs in AWS or GCP and peer them with application VPCs in the same cloud provider. VPC networking is managed on the **VPC Network** tab of the **Network Access** page.
```

Comment:
Is it worth saying right up front that you use VPC networks to lower your costs within a single cloud provider? I know it's in the third sentence and all... (Also is cross-provider VPC a thing, or is the idea of VPC to keep as much traffic as you can internal to a single provider?)